### PR TITLE
MC-1417 Updating documentation for MC 5.1

### PR DESCRIPTION
**Pull request by erosb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,15 +40,31 @@ you can browse Hazelcast Management Center using the URL: `http://host-ip:8080`.
 By default, the container automatically sizes the Java heap memory suitable to the specified resource limit or 
 available memory.
 
-### Management Center Default Context Path
+## Configuration options
+
+With from Management Center version 5.1, configuration options have been consolidated. System properties and environment variables
+can be used interchangeably. The complete reference of available settings is available in the [Management Center Reference Manual](https://docs.hazelcast.com/management-center/5.1/deploy-manage/system-properties). Below you can find example configurations
```
Comment (promanenko) [inline README.md:46]: `5.1` -> `latest` in the link

**Review by promanenko — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -44,15 +40,31 @@ you can browse Hazelcast Management Center using the URL: `http://host-ip:8080`.
 By default, the container automatically sizes the Java heap memory suitable to the specified resource limit or 
 available memory.
 
-### Management Center Default Context Path
+## Configuration options
```
Comment (JakeSCahill) [inline README.md:43]: This doc uses title case elsewhere. We should make sure that all headings are in the same case.

File: README.md [hunk 3]
```diff
@@ -44,15 +40,31 @@ you can browse Hazelcast Management Center using the URL: `http://host-ip:8080`.
...
+
+With from Management Center version 5.1, configuration options have been consolidated. System properties and environment variables
```
Comment (JakeSCahill) [inline README.md:45]:
```suggestion
In Management Center version 5.1 and later, system properties and environment variables
```

**Review by JakeSCahill — APPROVED** [review]
A couple of minor comments and suggestions:

- Should we avoid referring to IMDG now? In the docs we usually just use Hazelcast.
- $MANAGEMENT_CENTER_VERSION would be clearer than $MANAGEMENT_CENTER

**Comment by erosb** [comment]
> $MANAGEMENT_CENTER_VERSION would be clearer than $MANAGEMENT_CENTER

@JakeSCahill how about MC_VERSION?

**Comment by JakeSCahill** [comment]
> > $MANAGEMENT_CENTER_VERSION would be clearer than $MANAGEMENT_CENTER
> 
> @JakeSCahill how about MC_VERSION?

yes sure 😄

**Pull request merged**